### ISSUE TITLE
Logical Error in CycleGAN_Solution.ipynb

### ISSUE THREAD

**Issue by ahmedag91** [udacity/deep-learning-v2-pytorch#310]
Dear Team members
[CycleGAN_Solution.zip](https://github.com/udacity/deep-learning-v2-pytorch/files/6214797/CycleGAN_Solution.zip)
,

In the Discriminator class the last convolutional layer (conv5) should be as follows:
self.conv5 = conv(conv_dim*8, 1, 8, stride=1, padding=0, batch_norm=False)

This command mentioned above will produce and image with size of 1, 1, 1. That is unlike the command in the current notebook which produces an image with size of 1, 7, 7. The notebook does not cause any runtime error as the functions real_mse_loss and fake_mse_loss subtracts only a single value which does not cause any matrix/tensor mistmatch. To check out this issue try to see the size of out in the implemented forward function of the Discriminator class. Please, find the attached notebook with the new fixed bug. I did not run the whole notebook, but I checked this when I was implementing my own solution of the exercise.

BR,
Ahmed
